gui/Form1.cs

```diff
@@ -63,12 +63,29 @@ private void pickFolder_Click(object sender, EventArgs e)
 			}
 		}
 
+		private void IncrementProgress()
+		{
+			if (this.InvokeRequired)
+			{
+				this.Invoke(new Action(() => IncrementProgress()));
+			}
+			else
+			{
+				repackProgress.Value++;
+			}
+		}
+
 		private async void repackButton_Click(object sender, EventArgs e)
 		{
+			List<string> files = new List<string>();
+			foreach (var v in lvlsList.Items) files.Add(v.ToString());
```

Comment:
The compiler seems to be issuing a warning "Possible null reference argument for parameter 'item' in 'void List<string>.Add(string item)'." for this line. I'm guessing it just can't see that it's never null or is there actually the possibility it'll be null?